### ISSUE TITLE
Locality Aware Backup Docs imply that more than one locality can be default

### ISSUE THREAD

**Issue by msbutler** [cockroachdb/docs#12489]
Michael Butler (msbutler) commented:

In the Locality-aware backup [docs](https://www.cockroachlabs.com/docs/stable/take-and-restore-locality-aware-backups.html):

"
A locality-aware backup is specified by a list of URIs, each of which has a COCKROACH_LOCALITY URL parameter whose single value is either default or a single locality key-value pair such as region=us-east. At least one COCKROACH_LOCALITY must be the default
"

Only one COCKROACH_LOCALITY can be default.

@dt 

Jira Issue: DOC-2088

**Comment by kathancox** [comment]
@msbutler Yes, sorry the automated thing came along and marked as stale. I'm going mark this as an issue for our bug-bash style quick fixits for this Friday!
